core/lib/config/src/configs/eth_sender.rs

```diff
@@ -96,6 +106,10 @@ pub struct SenderConfig {
 
     /// The mode in which proofs are loaded, either from DB/GCS for FRI/Old proof.
     pub proof_loading_mode: ProofLoadingMode,
+
+    /// The mode in which we send pubdata, either Calldata, 1 Blob, or 2 Blobs/
+    /// If not specified, Calldata will be used
+    pub pubdata_sending_mode: Option<PubdataSendingMode>,
```

Comment:
it would be a bit more ergonomic to remove `Option` and use `.unwrap_or_default()` at the place of this creation.